src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The Gateway engine Pod has been renamed from `submariner` to `submariner-gateway`.
+* The helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the operator instead of `subctl`.
+* Each Submariner Pod now has it's own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric was renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy Service Discovery
+  only via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* `lighthouse-coredns` no longer relies on `Gateway` object to determine `localClusterID`.
+* Lighthouse uses `coredns-custom` configmap for AKS deployments.
+* The `strongswan` cable driver was removed.
+* Configuring Globalnet on a cluster is now optional with the `--globalnet` flag.
+* The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
+
```

Comment:
The following needs to be captured as well.
https://github.com/submariner-io/submariner-operator/issues/1126

Reply:
That should be part of 0.9.0-M2 release note, right?

Reply:
@sridhargaddam

Reply:
Do we have separate release notes for 0.9.0-M2? I don't think so. 
AFAICT the notes about `subctl validate ....` should be captured along with v0.9.0 release notes.

Reply:
m1 and m2 are internal dev milestones. Users should just care about 0.9.0

Reply:
I meant a separate commit in the same PR